src/lib/mongoBackend/MongoCommonRegister.cpp

```diff
@@ -82,7 +83,7 @@ static bool processSubscriptions
                                          trigs->attrL,
                                          mapSubId,
                                          trigs->reference,
-                                         trigs->format,
+                                         JSON,
```

Comment:
This means that format is "hardwired" to "legacy JSON" in NGSI9 notifications? Given that no alternative format has been defined in NGSIv2 for this kind of notifications :) I think is the right way of doing

However, maybe a comment like "// FIXME: this will be unhardwirer once ctx availability notification formats get defined for NGSIv2"